Header header

float64[] audio_feature
float64 audio_power
float64 audio_azimuth
float64[] audio_head_joints
string audio_cmd

float64 audio_wrist_rms
float64[] audio_wrist_mfcc

float64[] kinematics_ee_pos
float64[] kinematics_ee_quat
float64[] kinematics_jnt_pos
float64[] kinematics_jnt_vel
float64[] kinematics_jnt_eff
float64[] kinematics_target_pos
float64[] kinematics_target_quat

float64[] ft_force
float64[] ft_torque

float64[] vision_artag_pos
float64[] vision_artag_quat

float64[] vision_landmark_pos
float64[] vision_landmark_quat

float64[] vision_change_centers_x
float64[] vision_change_centers_y
float64[] vision_change_centers_z

float64[] pps_skin_left
float64[] pps_skin_right

float64[] fabric_skin_centers_x
float64[] fabric_skin_centers_y
float64[] fabric_skin_centers_z
float64[] fabric_skin_normals_x
float64[] fabric_skin_normals_y
float64[] fabric_skin_normals_z
float64[] fabric_skin_values_x
float64[] fabric_skin_values_y
float64[] fabric_skin_values_z





